packages/rest/src/router/routing-table.ts

```diff
@@ -238,6 +240,31 @@ export interface ResolvedRoute extends RouteEntry {
   readonly schemas: SchemasObject;
 }
 
+export class StaticRoute implements RouteEntry {
```

Comment:
Let's call this `StaticAssetsRoute`. The name "static route" can be also interpreted as a route that's static (e.g. does not accept any path parameters).